challenge_1_answer.py

```diff
@@ -0,0 +1,23 @@
+import pandas
+
+# Import csv into dataframe, extract dimensions and remove Box Dimensions string from dataframe
+df = pandas.read_csv("list_of_presents.csv")
+df[['Length', 'Width', 'Height']] = df['Box Dimensions'].str.extract('(\d{1,2})x(\d{1,2})x(\d{1,2})').astype(int)
```

Comment:
This line raises a `SyntaxWarning: invalid escape sequence '\d'`. The backslashes in regular expressions (like \d) could be interpreted as escape characters. It is not an issue in this case as you still get the correct answer. But in the future and to avoid this current raised warning issue. Convert the Regex to a Raw string `r'(\d{1,2})x(\d{1,2})x(\d{1,2})'` this will avoid an confusion and stop the warning that appears.